torch/onnx/_internal/exporter/_torchlib/ops/nn.py

```diff
@@ -0,0 +1,71 @@
+"""torch.ops.aten operators under the `core` module."""
+# mypy: disable-error-code="misc,arg-type,type-arg,valid-type,assignment,return-value,type-var,operator,no-untyped-def,index"
+# ruff: noqa: TCH001,TCH002
+# flake8: noqa
+
+from __future__ import annotations
+
+import math
+
+from onnxscript.onnx_opset import opset18 as op, opset20 as op20
+
+import torch
+from torch.onnx._internal.exporter._torchlib._tensor_typing import TReal
+from torch.onnx._internal.exporter._torchlib._torchlib_registry import onnx_impl
+
+
+_MATH_PI = math.pi
+
+
+aten = torch.ops.aten
+
+
+@onnx_impl(aten.gelu.default, trace_only=True)
+def aten_gelu(self: TReal, approximate: str = "none") -> TReal:
+    """gelu(Tensor self, *, str approximate='none') -> Tensor"""
+
+    if approximate == "tanh":
+        result = _aten_gelu_approximate_tanh(self)
+    else:
+        result = _aten_gelu_approximate_none(self)
+    return result
+
+
+@onnx_impl(aten.gelu.default, private=True)
```

Comment:
Use trace_only instead. I believe I have removed the private option already. If not we should remove the private tag from all implementations.

Reply:
In fact for this one we can remove the decorator.